.github/workflows/pr-gh-workflow-approve.yaml

```diff
@@ -0,0 +1,40 @@
+name: PR approve GH Workflows
+
+on:
+  pull_request_target:
```

Comment:
do we need this? or can be just pull_request? this can lead to some security issues